packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -92,6 +93,9 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
 
     const [travelMode, setTravelMode] = useRecoilState(travelModeState);
 
+     /** Indicate if the user has My Position selected */
+    const [hasMyPostion, setHasMyPosition] = useState(false);
```

Comment:
`has` can also indicate that the position is known by the app. How about `myPositionSelected`?

Reply:
Good point, renamed the use state ✅